src/GitHunter.Desktop.Shared/GitHunter.Desktop.Shared.csproj

```diff
@@ -2,6 +2,7 @@
 
     <PropertyGroup>
         <TargetFramework>net7.0</TargetFramework>
+        <RootNamespace>GitHunter.Desktop</RootNamespace>
```

Comment:
What does this part do?

Reply:
To use the same namespace for the module itself and the shared module